### DISCUSSION TITLE
Upgrade validators to ignore http:\\  https\\ errors

### DISCUSSION THREAD

**Discussion by HerzovanderWal** [INSPIRE-MIF/helpdesk-validator#859] [Ideas]
Hi there,

first: I miss the Category: Improve (proposal)
This is one.

 I'm providing data and metadata to use in boarder crossing issues that should be compliant to international rules. In order to do so we provide data and resources secure. Thats why we use https.

Yesterday i'm confronted with the fact that the validators still control that the EU-resources are provided with the http|:\\ protocol. Even now where the sources them selves are provided https://.  This means that we constantly have to downgrade our discriptions and connect to expensive and energy consuming redirects. Is it possible to configure the validators to by-pass these http(s) controls? Or when will https:// be accepted?

See also the not answered questions in this repossitory!!

Sample: In tagblock keywords PriorityDataset: https:// is not accepted in active URL https://inspire.ec.europa.eu/theme. The workarround is downgrading to http://...

If not we miss a green flag in the EU geodata portal. 

Herzo

**Comment by fabiovinci** [comment]
Dear @HerzovanderWal,

the reason why the validator checks that codelist values are provided in HTTP is that this type of URL acts as an identifier.

![image](https://user-images.githubusercontent.com/22212406/211312464-23750f40-97a7-4e04-be5f-fbdf70143e7c.png)

Could you please explain better the issue in the EU geodata portal?

**Reply by HerzovanderWal** [reply]
Thank you: Thats an eye opener. 
So in the definition of the URI is the protocol. 
May be it is more conveniënt to improve this definition by separating the protocol from the URI definition. 
Where do we need to go?
